auth-web/src/views/auth/home/IncorpOrRegisterView.vue

```diff
@@ -200,4 +290,24 @@ export default class IncorpOrRegisterView extends Vue {
       text-decoration: underline;
     }
   }
+
+  .tooltip {
+    background-color: transparent;
+    opacity: 1 !important;
+
+    .tooltip-content {
+      min-width: 30rem;
+      padding: 2rem;
+      font-size: 0.75rem;
```

Comment:
To make it easier to globally search for font sizes, we usually use a CSS variable here.

For this particular size, you'll have to add a new variable here:
https://github.com/bcgov/sbc-auth/blob/main/auth-web/src/assets/scss/theme.scss

Reply:
got it. Will do.

Reply:
Done